### PR TITLE
Add flat tire support & allow to change edge gesture sensitivity.

### PR DESCRIPTION
**Pull request by MagneFire**

### CODE REVIEW

File: src/utils/src/deviceinfo.cpp [hunk 1]
```diff
@@ -26,6 +26,24 @@ bool DeviceInfo::hasRoundScreen()
...
+double DeviceInfo::edgeBoundary()
+{
+#ifdef EDGE_BOUNDARY
+    return EDGE_BOUNDARY;
+#else
+    return 0.1;
+#endif
+}
+
+int DeviceInfo::hasFlatTire()
```
Comment (FlorentRevest) [inline src/utils/src/deviceinfo.cpp:38]: Given that this returns the actual  size of the flat tire, I believe that flatTireHeight would be a better name than hasFlatTire

File: src/utils/src/deviceinfo.h [hunk 2]
```diff
@@ -27,6 +27,8 @@ class DeviceInfo : public QObject
     Q_OBJECT
     Q_DISABLE_COPY(DeviceInfo)
     Q_PROPERTY(bool hasRoundScreen READ hasRoundScreen CONSTANT)
+    Q_PROPERTY(double  edgeBoundary READ edgeBoundary CONSTANT)
```
Comment (FlorentRevest) [inline src/utils/src/deviceinfo.h:30]: In this file you use double spaces before variable names for no apparent reason

Comment (MagneFire) [reply src/utils/src/deviceinfo.h:30]: Alignment was the initial reason. But once I added edgeBoundary it all fell out of place :smile:

Comment (MagneFire) [reply src/utils/src/deviceinfo.h:30]: Maybe change the name of this function to edgeSensitivity instead?

Comment (FlorentRevest) [reply src/utils/src/deviceinfo.h:30]: Maybe something like BorderGestureWidth ?
This isn't really a sensitivity and this is only an edge for the BorderGestureArea item.

Comment (MagneFire) [reply src/utils/src/deviceinfo.h:30]: oh, yeah even better!

**Comment by MagneFire** [comment]
I completely agree, this stuff should probably be in a configuration file...

**Pull request merged**